x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/metrics/metrics_template.tsx

```diff
@@ -111,94 +120,107 @@ export const MetricsTemplate = React.forwardRef<HTMLDivElement, { children: Reac
 
     const quickAccessItems = [...quickAccessItemsRef.current];
 
+    const showAddMetricsCallout =
+      dataStreamsStatus === 'success' && !hasMetrics(dataStreams) && renderMode.mode === 'page';
+    const addMetricsCalloutId: AddMetricsCalloutKey =
+      asset.type === 'host' ? 'hostMetrics' : 'containerMetrics';
+
     return (
-      <EuiFlexGroup
-        gutterSize="s"
-        direction="row"
-        css={css`
-          ${useEuiMaxBreakpoint('xl')} {
-            flex-direction: column;
-          }
-        `}
-        data-test-subj="infraAssetDetailsMetricChartsContent"
-        ref={ref}
-      >
-        <EuiFlexItem
-          grow={false}
+      <>
+        {showAddMetricsCallout && (
+          <>
+            <AddMetricsCallout id={addMetricsCalloutId} /> <EuiSpacer size="s" />
```

Comment:
Can you add a new line? I was about to suggest to remove the Fragment XD
```suggestion
            <AddMetricsCallout id={addMetricsCalloutId} />
            <EuiSpacer size="s" />
```